packages/atomic/cypress/integration/facets/segmented-facet/segmented-facet-selectors.ts

```diff
@@ -0,0 +1,78 @@
+export const segmentedFacetComponent = 'atomic-segmented-facet';
+export const SegmentedFacetSelectors = {
+  withId(id: string) {
```

Comment:
This style of selector vs the style of selector that other non-facet components use exists mostly to test facet dependencies. However, I think it makes sense for you to use it here like you just did, since we may want to support that eventually.